common/constants/shared.ts

```diff
@@ -30,6 +30,7 @@ export const observabilityPluginOrder = 6000;
 export const UI_DATE_FORMAT = 'MM/DD/YYYY hh:mm A';
 export const PPL_DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 export const PPL_INDEX_REGEX = /(search source|source|index)\s*=\s*([^|\s]+)/i;
+export const PPL_CONTAINS_TIMESTAMP_REGEX = /\|\s*.*\s*[<|<=|=|>=|>]\s*timestamp\([^\)]+\)/i;
```

Comment:
Do you need this timestamp regex in custom panel? This was added previously but the new requirement asks to not check if a query contains a date range anymore, I've already removed on my side.

Reply:
This is added here cause, I still need to remove time range from queries in "savedVisualization" data modal. 
1. If we are still allowing the users to add time range in query in event analytics, then I need to remove them explicitly in panels' visualizations.
2. If we are not allowing users to add time range in query in event analytics, then I'll remove this check. 
Please let me know if got something wrong here.

Reply:
Oh yes, I understand this point that date range is stored separately and only added in runtime. But, I would like to know what happens if the user adds a time range in the query? Do we explicitly remove? Or Do we throw an error toast saying "adding time range is not allowed in query"? If we have such checks I'll remove the checks from my side.